### PR TITLE
Guest Quiz session code dump

### PR DESCRIPTION
**Pull request by caioertai**
@jamiesonreinhard 

Close when finished.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -10,6 +10,10 @@
     resources :payments, only: :new
   end
 
+  namespace :guest do
+    resources :sessions, only: :create
+  end
```
Comment (caioertai) [inline config/routes.rb:15]: Needed to post from home

File: app/views/quiz_questions/_fields.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="list-group pt-4">
+      <%= fqq.association :answer,
+          collection: fqq.object.possible_answers,
+          wrapper: :radio_list_group,
+          as: :radio_buttons,
+          label: false %>
+    </div>
+  </li>
+  <hr class="mb-4">
+<% end %>
```
Comment (caioertai) [inline app/views/quiz_questions/_fields.html.erb:16]: To remove the repetition on how questions are displayed in quiz sessions (between guest and course)

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -16,6 +16,8 @@
   </div>
 </div>
 
+<%= render "guest/quiz_sessions/form", quiz_session: @intro_quiz_session %>
```
Comment (caioertai) [inline app/views/pages/home.html.erb:19]: Needs styling

File: app/views/guest/quiz_sessions/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<ol class="p-0">
+  <%= simple_form_for [:guest, quiz_session], html: { class: "quiz-answer" } do |f| %>
+    <%= render "quiz_questions/fields", form: f %>
+    <div class="text-right"><%= f.button :submit, "Finish quiz", class: "btn-primary btn-lg" %></div>
+  <% end %>
+</ol>
```
Comment (caioertai) [inline app/views/guest/quiz_sessions/_form.html.erb:6]: Annoying repetition from course form.... but maybe needed for now.

File: app/views/course/quiz_sessions/_form.html.erb [hunk 5]
```diff
@@ -1,21 +1,6 @@
...
-          <%= fqq.association :answer,
-              collection: fqq.object.possible_answers,
-              wrapper: :radio_list_group,
-              as: :radio_buttons,
-              label: false %>
-        </div>
-      </li>
-      <hr class="mb-4">
-    <% end %>
+    <%= render "quiz_questions/fields", form: f %>
```
Comment (caioertai) [inline app/views/course/quiz_sessions/_form.html.erb:3]: Using new partial for questions

**Review by jamiesonreinhard — COMMENTED** [review]
Reviewed all commits and used as reference to add commits to branch 93-new-intro-quiz. We can close this branch as all of this code has now been added to 93.

**Comment by jamiesonreinhard** [comment]
Do not merge - code will be committed with 93-new-intro-quiz

**Pull request closed without merging**